### ISSUE TITLE
Leaderboard should sort by any column

### ISSUE THREAD

**Issue by burnsfisher** [ac2cz/FoxServerSide#9]
AK7WS suggests (and I think it is a good idea) that one should be able to sort the leaderboard by different columns, including total, last 24 hours, last 7 days, and she suggests a new column for last 4 months so that newbies would have some reasonable chance of being seen.

My addition would be the ability to sort by grid square.  It would be nice to see which grid squares are well-represented and which poorly-represented.

The whole idea is of course to encourage people to collect telemetry, even people who are just starting.  This will be especially important when we get whole-orbit-data and we will want to encourage people in empty grid squares.

**Comment by ac2cz** [comment]
Is this about sorting the leaderboard?  Or changing how we determine who the top 10 are?  You can currently search for your call-sign and see how you are doing.  

With that said, I suspect that a 4 month view will still show the same stations in the top 10.  They just download more than anyone else every day...

**Comment by burnsfisher** [comment]
I'm not sure I catch your meaning.  What is the difference between sorting and how we determine the top 10?  I guess she would like to see the top N of the last 4 months.  I did point out that the top 10 will almost always continue to be the top (or at least the top 6 or 8 will if they continue to collect at the same rate) but at least a new contributor could see him/herself climbing rapidly rather than having to 'overcome' all the collected data from the past.

**Comment by lisa50469** [comment]
I've been tracking the FOX satellites for a while now, and one thing
that bothers me is the leaderboard.  It is CLEARLY biased toward those
that have been tracking for longer than I have been.

It's sorted by the 'Total Frames' which means those that have been
around longer remain a the top and can't be touched.  The site needs to
be updated to do a running total over a maximum of a few months.  That
would allow folks like me that can NO WAY EVER catch up to the leaders
to at least get close.

Right now I run a small system, and enjoy it, but it's a little
disturbing to see people with over 200K frames knowing they are
un-touchable.  IF/When I get to 200K frames, they will have over 400K

Put simply, the 'Total' is impossible, and the 'Last 7 Days' is not long
enough.  IF I were in charge of this, I'd keep the total count on the
screen, and add '4 Months' to the list and sort by that. 

This change might get more people interested because some people will
just give up because they have no chance to make the top 10.  Not me, I
don't mind being lower on the list.

This is just my opinion, I enjoy seeing how much data I can collect.

Lisa AK7WS

**Comment by ac2cz** [comment]
Thanks Lisa, it is a good suggestion but I have to think about it.  I agree it may be more fair, but it would also change the rules of the game for those who have been contributing for a while.  I agree that they are un-catch-able while they contribute, but some of the stations that were top of the leaderboard originally stopped regular contributions and have now been over taken.

Burns suggest alternative sorts and I will think about that too.  Perhaps "top newcomers" or "last 4 months" could be other views, but perhaps not the default view. 

So let me think about it some more and see what can be done.

**Comment by burnsfisher** [comment]
Lisa also sent me this:
The more I think about it, I think a year is fine for the leaderboard totals...
I'm not running my system right now due to the weather... My system is portable and not very good in the rain.
I'll let you (Chris) think about different ways to slice and dice the data.
